### PR TITLE
Get rid of services injectors in stream-view.ts

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: app/services/restream.ts [hunk 1]
```diff
@@ -69,26 +69,32 @@ export class RestreamService extends StatefulService<IRestreamState> {
...
+       * This determines whether the user can enable restream
+       * Requirements:
+       * - Has prime, or
+       * - Has a grandfathered status enabled
+       */
+      get canEnableRestream() {
+        const userView = this.getServiceViews(UserService);
+        return userView.isPrime || (userView.auth && this.state.grandfathered);
+      }
+    }
```
Comment (andycreeth) [inline app/services/restream.ts:84]: Can you explain why you're starting the pattern of defining the view handler inside the getter like this?

Comment (andycreeth) [reply app/services/restream.ts:84]: @holiber bump

Comment (holiber) [reply app/services/restream.ts:84]: I have only one method here and we don't export this class outside the file. So just don't see a big difference between declaring this class separately

**Comment by andycreeth** [comment]
Can you rebase this onto master?  It only changes frontend files so it's best if it goes straight to master.

**Comment by andycreeth** [comment]
Seems like there is now a merge conflict.

**Pull request merged**